server/controller/src/main/java/ai/starwhale/mlops/domain/upgrade/step/UpdateK8sImage.java

```diff
@@ -21,7 +21,7 @@
 import ai.starwhale.mlops.domain.upgrade.bo.Upgrade;
 import ai.starwhale.mlops.exception.SwProcessException;
 import ai.starwhale.mlops.exception.SwProcessException.ErrorType;
-import ai.starwhale.mlops.schedule.k8s.K8sClient;
```

Comment:
How about add the @ConditionalOnProperty(value = "sw.scheduler", havingValue = "k8s") so that there is unnecessary to upgrade in a docker environment.